.github/workflows/build.yaml

```diff
@@ -0,0 +1,19 @@
+name: Lzycat Build 
+
+on: 
+  workflow_dispatch:
+  pull_request: 
+    branches: 
+      - main 
+
+jobs:
+  runtest: 
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout the repository
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Running docker scripts
+      run: ./run_docker.sh
```

Comment:
Can you also update the `./run_docker.sh`

Need to enable the docker build script

Thanks @munirapp

Reply:
https://github.com/lazycatlabs/blog.lazycatlabs/pull/29/commits/d81a8cb4c2dacc249b70bab02d3a739f3d6f7125